src/mumble/LookConfig.cpp

```diff
@@ -171,6 +187,8 @@ void LookConfig::load(const Settings &r) {
 		}
 	}
 	
+	updateFontInfo(r);
```

Comment:
What is this doing? It appears somewhat lonely in the code, so I think there should be a comment